src/Fixer/Phpdoc/PhpdocToCommentFixer.php

```diff
@@ -111,6 +130,10 @@ protected function createConfigurationDefinition(): FixerConfigurationResolverIn
                 ->setAllowedTypes(['array'])
                 ->setDefault([])
                 ->getOption(),
+            (new FixerOptionBuilder('allow_before_return_statement', 'Whether to allow PhpDoc before return statement.'))
+                ->setAllowedTypes(['bool'])
+                ->setDefault(false) // @TODO 4.0: set to `true`
```

Comment:
Was it agreed somewhere about this?

Reply:
It was included in [my comment](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7402#pullrequestreview-1726138009) after our internal talk on the chat.